tests/hotplug/rolloutstrategy.go

```diff
@@ -63,6 +65,7 @@ var _ = Describe("[Serial][sig-compute]VM Rollout Strategy", decorators.SigCompu
 			vm := libvmi.NewVirtualMachine(vmi, libvmi.WithRunning())
 			vm, err := virtClient.VirtualMachine(vm.Namespace).Create(context.Background(), vm)
 			Expect(err).NotTo(HaveOccurred())
+			libwait.WaitUntilVMIReady(vmi, console.LoginToCirros)
```

Comment:
My bad!
different failure:
```suggestion
			Eventually(func() error {
				vmi, err = kubevirt.Client().VirtualMachineInstance(vm.Namespace).Get(context.Background(), vm.Name, &metav1.GetOptions{})
				return err
			}, 120*time.Second, 1*time.Second).ShouldNot(HaveOccurred())
			libwait.WaitUntilVMIReady(vmi, console.LoginToCirros)
```